src/kernel/include/shared/cos_config.h

```diff
@@ -10,7 +10,7 @@
 
 // After how many seconds should schedulers print out their information?
 #define SCHED_PRINTOUT_PERIOD  30
-//#define COMPONENT_ASSERTIONS   1 // activate assertions in components?
+#define COMPONENT_ASSERTIONS   1 // activate assertions in components?
```

Comment:
@gedare 's statement is generally correct.

However, here we do want the default to be that assertions are enabled.  If this were to be PRed correctly, this would be a separate PR that is a bug fix.